.quickstart/quickstart.yml

```diff
@@ -5,6 +5,7 @@ dbt_versions: ">=1.3.0 <2.0.0"
 
 table_variables:
   using_schedules:
+    - audit_log
```

Comment:
Do I need to include this you can use schedules without the audit log? Same for schedule_holiday now.

Reply:
Hmmm good question. We may need to adjust these quickstart variables to match the tables more accurately. We can discuss after my initial review.

Reply:
Based off our discussion let's move forward with removing audit_log and schedule_history from the schedule variable and have them be their own. We can then rely on the combo of variables to control the enablement/disablement